src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer start  working.");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Bulldozer stop working.");
+
```

Comment:
For the future, remove the last empty line in all classes/interfaces/methods where it is present